lms/djangoapps/instructor/views/api.py

```diff
@@ -2563,6 +2563,24 @@ def export_ora2_data(request, course_id):
     return JsonResponse({"status": success_status})
 
 
+@transaction.non_atomic_requests
+@require_POST
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+@common_exceptions_400
+def export_ora2_summary(request, course_id):
```

Comment:
Needs tests added, like [TestOra2ResponsesInstructorTask](https://github.com/edx/edx-platform/blob/3b65081fabc505ef3e63256d7152882f1454f826/lms/djangoapps/instructor_task/tests/test_tasks.py#L658), [test_submit_ora2_request_task](https://github.com/edx/edx-platform/blob/3b65081fabc505ef3e63256d7152882f1454f826/lms/djangoapps/instructor_task/tests/test_api.py#L310-L318), and the tests mentioning [`export_ora2_data` in `instructor.tests.test_api`](https://github.com/edx/edx-platform/blob/871bc39b641fc83756f5234f8779b123060be343/lms/djangoapps/instructor/tests/test_api.py)